generator/config/config.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+use MongoDB\Aggregation\Operator;
+use MongoDB\Query\Operator as QueryOperator;
+use MongoDB\Aggregation\Stage;
+use MongoDB\CodeGenerator\FactoryClassGenerator;
+use MongoDB\CodeGenerator\ValueClassGenerator;
+
+$src = __DIR__ . '/../../src';
+
+return [
+    // Stages
+    [
+        'configFile' => __DIR__ . '/stages.yaml',
+        'generatorClass' => ValueClassGenerator::class,
+        'namespace' => MongoDB\Aggregation\Stage::class,
+        'filePath' => $src . '/Aggregation/Stage/',
+        'interfaces' => [Stage::class],
+        'classNameSuffix' => 'Stage',
+    ],
+    /*
+    // Stage codec
+    [
+        // Stage converters
+        'configFile' => __DIR__ . '/stages.yaml',
+        'generatorClass' => ConverterClassGenerator::class,
```

Comment:
I'm going to leave the codec generator for now and start writing a generic codec that can encode any operator: less codec objects to instantiate and maybe easier to write.